task01-html-css/task01.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<!-- 
+  All Rights Reserved
+  This software is property information of AKUREY S.A.
+  Contact us at contact@akurey.com
+  @summary Html of task 01, grid with 04 images
+-->
+<html>
+    <head>
+        <link rel="stylesheet" href="task01.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@400&display=swap" rel="stylesheet">
+        <title>One Piece</title>
+    </head>
+    <body>
+        <div class="grid-container">
+           <div class="grid-item" id="img-monkey"><div class="img-text">Monkey D. Luffy</div></div>
+           <div class="grid-item" id="img-portgas"><div class="img-text">Portgas D. Ace</div></div>
+           <div class="grid-item" id="img-roronoa"><div class="img-text">Roronoa Zoro</div></div>
+           <div class="grid-item" id="img-nico"><div class="img-text">Nico Robin</div></div>
```

Comment:
Let's change the div statement for another one, like <p> or <span>.
DIV is for grouping elements, so it would be better to use another statement for this case.